docs/en/cloud/security/ip-egress-traffic-list.md

```diff
@@ -0,0 +1,50 @@
+---
+slug: /en/manage/security/ip-egress-traffic-list
+sidebar_label: IP Egress Traffic List
+title: IP Egress Traffic List
+---
+
+## Integrations could require egress access
+
+If you are using integration like the MySQL or PostgreSQL Engine, It's possible that you need to authorize ClickHouse Cloud to access to your instances.
+
+You could use this list of public IPs in `firewalls` or `Authorized networks` in GCP or in `Security Groups` in AWS or in any other infrastrucutre egrees management system you are using.
```

Comment:
Can we add screenshots to help users to find these?

Reply:
I have this one for GCP, do we want one for AWS too ?
<img width="586" alt="Screenshot 2022-12-20 at 12 25 03" src="https://user-images.githubusercontent.com/6648062/208928123-570e3a89-82e5-4d17-95de-0a45289e68f5.png">

Reply:
Should there be a note about not allowing these public IPs if using AWS  PrivateLink, or would the person managing this know to ignore this if they are using PrivateLink? (I guess they would know that, right?)

Reply:
Hey @DanRoscigno , that is a good catch, I think we should add something about it, we could just do a small `note` about it?